### PR TITLE
Remaps Lava Pinnance

### PR DESCRIPTION
**Pull request by Erikafox**
## About The Pull Request
Slightly remaps and readds lava pinnance. It has more of a footprint now, *and* an implied story. Snowflakes a few items to sell it.

StrongDMM-2024-09-15 20 07 18
StrongDMM-2024-09-15 20 07 21

## Why It's Good For The Game
Not entirely sold on this one being good, but it's a nice fluff piece. 

## Changelog

:cl:
add: Joyriding scientists have reported crashing their research craft on a nearby lavaworld. They're probably dead now, but some of their stuff has to be okay.
/:cl:

### CODE REVIEW

File: code/modules/cargo/packs/medical.dm [hunk 1]
```diff
@@ -48,6 +48,13 @@
 	small_item = TRUE
 	contains = list(/obj/item/storage/firstaid/radiation)
 
+/datum/supply_pack/medical/ifak
```
Comment (thgvr) [inline code/modules/cargo/packs/medical.dm]: This seems to be an unrelated change to the ruin.

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 2]
```diff
@@ -187,12 +187,20 @@
...
 	item_state = "morphen"
 	list_reagents = list(/datum/reagent/medicine/morphine = 10)
 
+/obj/item/reagent_containers/hypospray/medipen/painkiller
+	name = "painkiller medipen"
+	desc = "An proprietary blend of painkillers, for keeping you up just long enough to bandage something."
+	icon_state = "painpen"
+	base_icon_state = "painpen"
+	item_state = "painpen"
+	list_reagents = list(/datum/reagent/medicine/mine_salve = 2, /datum/reagent/medicine/morphine = 8)
```
Comment (thgvr) [inline code/modules/reagents/reagent_containers/hypospray.dm]: ...this will just put you to sleep.

File: code/modules/cargo/packs/medical.dm (hunk 1 above)
Comment (Erikafox) [reply code/modules/cargo/packs/medical.dm]: tangentially related item from adhd-coding. can cut
